recipes/xorg-libximerna/build.sh

```diff
@@ -0,0 +1,7 @@
+set -ex
+
+export ACLOCAL="aclocal -I ${PREFIX}/share/aclocal"
+./autogen.sh --prefix ${PREFIX} --disable-staticc
+make install
+
+set +ex
```

Comment:
This command doesn't do anything since the shell is just going to exit anyway.

Reply:
I guess previously with travis that wasn't so true. But maybe I am wrong. I just removed this line.